src/H5Fsuper_cache.c

```diff
@@ -433,24 +433,38 @@ H5F__cache_superblock_deserialize(const void *_image, size_t H5_ATTR_NDEBUG_UNUS
     if (H5F__superblock_prefix_decode(sblock, &image, udata, FALSE) < 0)
         HGOTO_ERROR(H5E_FILE, H5E_CANTDECODE, NULL, "can't decode file superblock prefix")
 
+    const uint8_t *image_end = _image + len - 1;
```

Comment:
Looks like the -Werror builds are failing due to arithmetic on a void pointer here. Suggest changing `_image` to `image`. Also, the clang format check was a little grumpy.

Reply:
Codespell's also complaining about the RELEASE.txt entry

Reply:
Fixed